packages/mdc-segmented-button/segment/_segment-theme.scss

```diff
@@ -0,0 +1,147 @@
+//
+// Copyright 2020 Google Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+
+// Theme: Variables, functions, and mixins required to theme component segment
+
+@use "@material/feature-targeting/functions" as feature-targeting-functions;
+@use "@material/feature-targeting/mixins" as feature-targeting-mixins;
+@use "@material/theme/variables" as theme-variables;
+@use "@material/theme/custom-properties";
+
+$on-surface: theme-variables.$on-surface;
+$surface: theme-variables.$surface;
+$primary: theme-variables.$primary;
+
+$unselected-ink-color: rgba($on-surface, 0.6);
+$unselected-outline-color: rgba($on-surface, 0.12);
+$unselected-container-fill-color: rgba($surface, 1);
+$selected-ink-color: rgba($primary, 1);
+$selected-outline-color: $unselected-outline-color;
+$selected-container-fill-color: rgba($primary, 0.08);
+
+$height: 36px;
+$min-width: 48px;
+$horizontal-padding: 12px;
+$border-width: 1px;
+$border-radius: 4px;
+$icon-width: 24px;
+$icon-font-size: 18px;
+$label-padding: 6px;
+
+@mixin unselected-ink-color($color, $query: feature-targeting-functions.all()) {
```

Comment:
Add documentation to all theme mixins following [SassDoc](http://sassdoc.com/) format.

Reply:
Is this just on the theme mixins or also in _segment.scss and _segmented-button.scss?

Reply:
Having docs for theme mixin is very useful. Most of the mixins in core style module files are structural related and used for grouping.